build.go

```diff
@@ -1,23 +1,25 @@
 package thing
 
 import (
-	"errors"
 	"fmt"
+	"log"
 	"os/exec"
 	"strings"
 )
 
 type Runner struct {
-	test    bool
-	History []string
-	Output  string
+	test      bool
+	History   []string
+	Output    string
+	updated   bool
+	installed bool
 }
 
 func NewRunner() *Runner {
 	return &Runner{}
 }
 
-func NewTestRunner() *Runner {
+func TestNewRunner() *Runner {
```

Comment:
Let's keep this as `NewTestRunner`, because our _runner_ is either a real Runner or a test Runner, isn't it? So NewRunner() creates a new Runner, and NewTestRunner() creates a new test Runner, which makes sense.

Reply:
That makes sense, I do not recall how the naming drifted to TestNewRunner. I noticed it on the first test case yesterday and thought it was something we changed. I'll keep it as we originally had it.